company/okrs/2020_q1.md

```diff
@@ -101,16 +101,16 @@
 ### Core services
 
 1. **Increase adoption of search at customers**
-   1. Write at least 2 pieces of technical documentation (docs, walkthroughs, blog posts) about search functionality.
-   1. 99th percentile of time to first result is below 3s across all search types.
-   1. At least four customers are running structural searches and metrics to validate this KR are tracked in [pings](https://docs.sourcegraph.com/admin/pings).
+   1. Write at least 2 pieces of technical documentation (docs, walkthroughs, blog posts) about search functionality. => (i) Structural search [blog post](https://about.sourcegraph.com/blog/going-beyond-regular-expressions-with-structural-code-search) (ii) Revamped [search syntax docs](https://docs.sourcegraph.com/user/search/queries#keywords-all-searches).
+   1. 99th percentile of time to first result is below 3s across all search types. => Added latency to track all search types via pings as of 3.14.
+   1. At least four customers are running structural searches and metrics to validate this KR are tracked in [pings](https://docs.sourcegraph.com/admin/pings). => Evidence of usage at existing and prospective customers (three confirmed).
```

Comment:
I put the tally here but didn't really want to. I think the OKR should be approximate about this sort of data (`yes/no` it is achieved, or a range like "yes, greater than or equal to 4"). We're not tracking usage long enough to give an accurate tally here either.